plugins/task/src/components/internal/NewTaskForm.svelte

```diff
@@ -28,32 +27,37 @@
 
   export let title: string
   let message: string = ''
-  export let space: Ref<Space>
+  export let space: Ref<Space> | undefined
+  export let spaces: Space[] | undefined
   let object = {} as any
 
+  let selectedSpaceIdx = 0
```

Comment:
SpacesCombo? Same code here.

Reply:
Also I think we should not show combo in case there is just one space available. Thoughts?

Reply:
Yeah, good idea